seeders/courses.seeder.js

```diff
@@ -12,13 +12,13 @@ const data = async () => {
   return [
     {
       title: "Programming1",
```

Comment:
This is probably more relevant on the open database PR, but we should probably do another pass over our seed data to make sure things are formatted correctly. I can't think of a reason why we'd leave out the space here?

Reply:
This was already addressed in a previous commit.